src/components/FilterTop.js

```diff
@@ -374,7 +373,7 @@ function Filter(props) {
   );
 }
 
-const getVerificationStatus = (active, approved) => {
+export const getVerificationStatus = (active, approved) => {
```

Comment:
If this is a common utility being shared across components, it should be moved to a common location (e.g. under `src/utils/`). We shouldn't import a function from another component like this.